source/ui/BioXAS/BioXASControlEditor.h

```diff
@@ -63,7 +67,7 @@ public slots:
 	/// Sets whether the control's name is used for the title text.
 	void setUseControlNameAsTitle(bool useName);
 	/// Sets the value.
-	virtual void setValue(double newValue);
+	virtual void setValue(AMNumber newValue);
```

Comment:
I use AMNumbers here because in theory it's possible for the editor value to be invalid. I'm mostly thinking of the cases where no control is set, or when a control has been set but isn't measurable.